autogen/main/cluster.tf.tmpl

```diff
@@ -185,6 +185,11 @@ resource "google_container_cluster" "primary" {
     horizontal_pod_autoscaling {
       disabled = !var.horizontal_pod_autoscaling
     }
+
+    gcp_filestore_csi_driver_config {
```

Comment:
I see a conflit with autopilot, so we will need to move this into the `% if autopilot_cluster != true %}` block

https://github.com/hashicorp/terraform-provider-google/blob/d3ab68df17134e96ad8f14548a1f68f45a0eeae1/google/resource_container_cluster.go#L239